test/acceptance/contour.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+var assert = require('assert');
+var testHelper = require('../helper');
+
+
+describe('contour', function () {
+
+    var QUERY_AIRBNB_ROOMS = 'select * from airbnb_rooms';
+
+    var analysisDefinition = {
+        type: 'contour',
+        params: {
+            source: {
+                type: 'source',
+                params: {
+                    query: QUERY_AIRBNB_ROOMS
+                }
+            },
+            column: 'price',
+            buffer: 0.2,
+            method: 'barymetric',
+            class_method: 'quantiles',
+            steps: 7,
+            resolution: 250
+        }
+    };
+
+    it('basic contour test', function (done) {
+        testHelper.createAnalyses(analysisDefinition, function (err, analysisResult) {
+            if (err) {
+                return done(err);
+            }
+
+            testHelper.getRows(analysisResult.getQuery(), function (err, rows) {
+                if (err) {
+                    return done(err);
+                }
+                assert.equal(rows.length, analysisDefinition.params.steps);
```

Comment:
It'd be nice assert if `cartodb_id` and `the_geom` are in the analysis' output. Like we did in `gravity`